PatrickGBecker/cerulean-media
# Feature/build

### Changes

* Adds helpers for static props
* Adds try-catch blocks to API calls for break protection
* Replaces fetch fn's with staticProp fns
* Moves API folder back into pages directory
* Adds 500 response if api fails
* Adds type for ApiError response
* Some files look heavily edited but my code formatter just ran on save

### How to test

* in your terminal run `git fetch` 
* `git checkout feature/build`
* Make sure your .env key is set to `NEXT_PUBLIC_BASE_URL=http://localhost:3000`
* run `npm run build`
* run `npm run start`

### When deploying

* make sure your `NEXT_PUBLIC_BASE_URL` is not local host but whatever the vercel deployment url is
